homeassistant/components/drop_connect/translations/en.json

```diff
@@ -0,0 +1,45 @@
+{
+    "config": {
+        "abort": {
+            "not_supported": "Configuration for DROP is through MQTT discovery. Use the DROP Connect app to connect your DROP Hub to your MQTT broker."
+        },
+        "step": {
+            "confirm": {
+                "title": "Confirm Association",
```

Comment:
```suggestion
                "title": "Confirm association",
```

Reply:
Changed in cd7cf47e119dfc63b31916c82b92070e5b4c9660.